analyzers/its/Directory.Build.targets

```diff
@@ -52,8 +52,8 @@
       <!-- Remove all previously added analyzers, except built-in SourceGenerators -->
       <Analyzer Remove="@(Analyzer)" Condition="!$([System.Text.RegularExpressions.Regex]::IsMatch('%(Filename)', '(Generator|Generators|SourceGeneration|SourceGenerators)$'))"/>
       <!-- Add the SonarAnalyzer analyzer DLLs -->
-      <Analyzer Include="$(BinariesFolder)\SonarAnalyzer*.dll" />
-      <Analyzer Include="$(BinariesFolder)\Google.Protobuf.dll" />
+      <Analyzer Include="$(BinariesFolder)\SonarAnalyzer.*\SonarAnalyzer*.dll" />
+      <Analyzer Include="$(BinariesFolder)\SonarAnalyzer.*\Google.Protobuf.dll" />
```

Comment:
On a first look, this should (by-accident) pick up the styling rules too => we'll need to move styling rules into the same structure to be regular.

Reply:
Ah, the `Internal*` prevents it from being picked up in the current implementation. That's not very nice (see the next comment)